calendarLibrary/src/main/java/com/amy/monthweek/materialcalendarview/MonthWeekMaterialCalendarWithSwipeView.java

```diff
@@ -155,6 +155,10 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
                 recyclerViewTop,
                 getMeasuredWidth(),
                 recyclerViewTop + getMeasuredHeight()-finalWeekModeHeight);
+        mRecyclerView.layout(0,
```

Comment:
[Q] @YuriPopiv I can't help but wonder if this should be `.layout(recyclerViewTop, getMeasuredWidth(),...` because when the screen first loads the list is right against the top of the calendar.

<img width="319" alt="image" src="https://user-images.githubusercontent.com/64619773/80787706-99e81580-8bca-11ea-8b5a-aab97c579d3e.png">